db/schema.rb

```diff
@@ -19,10 +19,10 @@
     t.string   "title"
     t.datetime "due_at"
     t.datetime "happens_at"
+    t.integer  "user_id"
     t.string   "slug"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer  "user_id"
```

Comment:
Weird. Master already have user_id on Choogles. Hope we don't see anymore of these in the future :rabbit: